agent/s3_downloader_test.go

```diff
@@ -11,12 +11,12 @@ func TestS3DowloaderBucketPath(t *testing.T) {
 	t.Parallel()
 
 	s3Uploader := NewS3Downloader(logger.Discard, S3DownloaderConfig{
```

Comment:
Not relevant to this PR but I noticed this in passing through this whole file..
```suggestion
	s3Downloader := NewS3Downloader(logger.Discard, S3DownloaderConfig{
```

Reply:
oh nooooo

Reply:
will fix ahahaha